foundation-extensions/mobile-core/signals/signal-api-reference.md

```diff
@@ -10,37 +10,62 @@ To get the version of the Signal extension, use the following code sample:
 {% tab title="Android" %}
 #### Java
 
+**Syntax**
+
+```java
+public static String extensionVersion();
+```
+
+**Example**
+
 ```java
 String signalExtensionVersion = Signal.extensionVersion();
 ```
 {% endtab %}
 
-{% tab title="iOS \(AEP 3.x\)" %}
+{% tab title="iOS (AEP 3.x)" %}
+**Syntax**
+
+```swift
+public static let extensionVersion
+```
+
+**Examples**
+
 **Swift**
 
 ```swift
 let version = Signal.extensionVersion
 ```
 
-**Objective C**
+**Objective-C**
 
 ```objectivec
 NSString *version = [AEPMobileSignal extensionVersion];
 ```
 {% endtab %}
 
-{% tab title="iOS \(ACP 2.x\)" %}
-**Objective C**
+{% tab title="iOS (ACP 2.x)" %}
+**Syntax**
 
 ```objectivec
-NSString *signalExtensionVersion = [ACPSignal extensionVersion];
+(nonnull NSString*) extensionVersion;
 ```
 
+**Examples**
+
 **Swift**
 
 ```swift
 var signalExtensionVersion  = ACPSignal.extensionVersion()
 ```
+
+**Objective-C**
+
+```objectivec
+NSString *signalExtensionVersion = [ACPSignal extensionVersion];
+```
+
```

Comment:
Can we include the registerExtension API too? Example https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#registerextension